sources/UFS

```diff
@@ -348,8 +341,12 @@ Copyright (c) 1988-1995, 2000, 2021-2022 by Venue & Xerox Corporation.
 (\UFSGenerateFiles
   [LAMBDA (FDEV PATTERN DESIREDPROPS OPTIONS)
 
+    (* ;; "Edited 27-Mar-2022 15:55 by rmk: Use the EXTENSION and VERFSION in the pattern instead of the inherited defaults")
+
+    (* ;; "rmk; Use the EXTENSION and VERFSION in the pattern instead of the inherited defaults")
```

Comment:
comment repeated ?

Reply:
Unfortunately, not exactly.  The rmk in the second one ends in a semicolon, not a colon, so probably the EDITDATE code didn't recognize it as an initials-comment expression to be dated.  So an error in my original type-in.  The EDITDATE could be loosened to strip any punctuation (or none) off a comment, to see if what's left is the user's initials.

But an underlying problem is that exiting SEDIT with ^X leaves the window open but without the date modification that then happens on the way out.  So the error isn't visible unless the window is actually closed and the function is edited again.  It would be nice if the edit-date could cause a still open window to be redisplayed.